src/integrationTest/groovy/wooga/gradle/unity/UnityIntegrationSpec.groovy

```diff
@@ -135,7 +136,7 @@ abstract class UnityIntegrationSpec extends IntegrationSpec {
 
         mockUnityFile = createFile("fakeUnity.bat", unityMainDirectory)
         mockUnityFile.executable = true
-        if (windows) {
+        if (PlatformUtils.windows) {
```

Comment:
You could have used a `static` import. Then the Filechange would not be that big.

Reply:
That's true. I admit I rarely use static imports. I will need to think about it.